flocker/node/script.py

```diff
@@ -438,7 +439,7 @@ class BackendDescription(PClass):
     # out.
     needs_cluster_id = field(type=bool, mandatory=True)
     # Config "dataset" keys required to initialize this backend.
-    required_config = field(type=set, mandatory=True, initial=set())
+    required_config = pset_field(unicode)
```

Comment:
More importantly, I think we need an initial value, otherwise we'll break for old drivers.